README.md

```diff
@@ -0,0 +1,16 @@
+Symfony docker
+======
+
+Usage
+------
+
+Run `docker-compose up -d`
+
+Choosing symfony version
+------
+
+To choose specific symfony version define `SYMFONY_VERSION` env variable for first run.
+
+Like:
+
+`SYMFONY_VERSION=3.4 docker-compose up -d`
```

Comment:
Can you add a note about the stability flag too?

Reply:
done. And I noticed we should merge a little fix first (#25)